core/tests/protractor_utils/ExplorationEditorStatsTab.js

```diff
@@ -30,9 +31,18 @@ var ExplorationEditorStatsTab = function() {
     return numPassersby.getText();
   };
 
+  var _getInitIssueText = function() {
+    return firstIssueElement.getText();
+  };
+
   this.expectNumPassersbyToBe = function(numPassersby) {
     expect(_getNumPassersby()).toMatch(numPassersby);
   };
+
+  this.clickInitIssue = function() {
+    expect(_getInitIssueText()).toMatch('Issue 1');
```

Comment:
Hmm, I don't like that `'Issue 1'` is hard-coded. Please instead accept it as an argument.

this.clickInitIssue = function(expectedIssueText) {
  expect(_getInitIssueText()).toMatch(expectedIssueText);
  firstIssueElement.click();
};

Reply:
Done.